aws/src/verify_account/app.py

```diff
@@ -0,0 +1,158 @@
+import re
+import logging
+import boto3
+from botocore.exceptions import ClientError
+import json
+from datetime import datetime
+import sys
+import os
+
+from m59utils import util_get_output_value
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+send_from_email_address = os.environ["send_from_email_address"]
+aws_region = os.environ["region_name"]
+domain_name = os.environ["domain_name"]    
```

Comment:
This is a little weird to specify in the environment since we never expect it to change.  Maybe have a default so it can be overridden if necessary?  i.e.

```
domain_name = os.environ["domain_name"] or meridian_domain_name
```

then specify meridian_domain_name in a shared file, maybe m59utils.py?